pkg/db-repo/change_history_manager.go

```diff
@@ -0,0 +1,236 @@
+package db_repo
+
+import (
+	"fmt"
+	"github.com/applike/gosoline/pkg/cfg"
+	"github.com/applike/gosoline/pkg/mon"
+	"github.com/jinzhu/gorm"
+	"reflect"
+	"strings"
+)
+
+type changeHistoryManagerSettings struct {
+	ChangeAuthorField string `cfg:"change_author_column"`
+	TableSuffix       string `cfg:"table_suffix" default:"history"`
```

Comment:
has to be default pluralized right? because this pluralization stuff is also part of gosoline. otherwise the default settings wont work?

Reply:
also currently thinking about, what happens when someone by accident has a table with real data which name matches the history table pattern. then it will be updated right? even if this would happen by accident? 

might be a problem for the future

Reply:
👍  I'll change the default to `history_entries`

Reply:
The name conflict is a problem for later. I doubt it will happen.